bundle/edu.gemini.dataman.app/src/main/scala/edu/gemini/dataman/app/GsaPostAction.scala

```diff
@@ -1,7 +1,9 @@
 package edu.gemini.dataman.app
 
 import edu.gemini.dataman.core._
-import edu.gemini.dataman.query.{GsaQaUpdateQuery, GsaResponse}
+import edu.gemini.gsa.core.{QaResponse, QaRequest}
+
+import edu.gemini.gsa.query.{GsaResponse, GsaAuth, GsaQaUpdateQuery, GsaHost}
```

Comment:
In the new bundle you preserve the `core` vs `query` split that the data manager has.  We might want to rearrange it a bit thinking about which parts are considered part of the public API -- in the data manager there is no public API and nothing is exported but this bundle is being made for use from multiple bundles.  Or perhaps we just dump it all into one package?  There's not much to it after all.

I was looking for the bundle's `build.sbt` to see what is exported vs. not exported but it's not part of this PR?

`GsaHost` moves into `query` from `core` was that intentional?